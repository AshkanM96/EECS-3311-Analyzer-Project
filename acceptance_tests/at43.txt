greater_than
boolean_constant(true)
boolean_constant(true)
type_check
reset
greater_than
integer_constant(8)
integer_constant(4)
type_check
evaluate
reset
greater_than
integer_constant(4)
integer_constant(4)
type_check
evaluate
reset
greater_than
integer_constant(2)
integer_constant(4)
type_check
evaluate
reset
less_than
integer_constant(8)
integer_constant(4)
type_check
evaluate
reset
less_than
integer_constant(4)
integer_constant(4)
type_check
evaluate
reset
less_than
integer_constant(2)
integer_constant(4)
type_check
evaluate
