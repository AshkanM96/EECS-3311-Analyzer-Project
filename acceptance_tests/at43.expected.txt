  Expression currently specified: ?
  Report: Expression is initialized.
->greater_than
  Expression currently specified: (? > nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True > ?)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True > True)
  Report: OK.
->type_check
  Expression currently specified: (True > True)
  Report: (True > True) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->greater_than
  Expression currently specified: (? > nil)
  Report: OK.
->integer_constant(8)
  Expression currently specified: (8 > ?)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (8 > 4)
  Report: OK.
->type_check
  Expression currently specified: (8 > 4)
  Report: (8 > 4) is type-correct.
->evaluate
  Expression currently specified: (8 > 4)
  Report: True
->reset
  Expression currently specified: ?
  Report: OK.
->greater_than
  Expression currently specified: (? > nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (4 > ?)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (4 > 4)
  Report: OK.
->type_check
  Expression currently specified: (4 > 4)
  Report: (4 > 4) is type-correct.
->evaluate
  Expression currently specified: (4 > 4)
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->greater_than
  Expression currently specified: (? > nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: (2 > ?)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (2 > 4)
  Report: OK.
->type_check
  Expression currently specified: (2 > 4)
  Report: (2 > 4) is type-correct.
->evaluate
  Expression currently specified: (2 > 4)
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->less_than
  Expression currently specified: (? < nil)
  Report: OK.
->integer_constant(8)
  Expression currently specified: (8 < ?)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (8 < 4)
  Report: OK.
->type_check
  Expression currently specified: (8 < 4)
  Report: (8 < 4) is type-correct.
->evaluate
  Expression currently specified: (8 < 4)
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->less_than
  Expression currently specified: (? < nil)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (4 < ?)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (4 < 4)
  Report: OK.
->type_check
  Expression currently specified: (4 < 4)
  Report: (4 < 4) is type-correct.
->evaluate
  Expression currently specified: (4 < 4)
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->less_than
  Expression currently specified: (? < nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: (2 < ?)
  Report: OK.
->integer_constant(4)
  Expression currently specified: (2 < 4)
  Report: OK.
->type_check
  Expression currently specified: (2 < 4)
  Report: (2 < 4) is type-correct.
->evaluate
  Expression currently specified: (2 < 4)
  Report: True
